src/DbUtils.php

```diff
@@ -1821,6 +1821,7 @@ public function getUserName($ID, $link = 0, $disable_anon = false)
                     $user_params = [
                         'id'                 => $ID,
                         'user_name'          => $username,
+                        'login'              => $data['name'],
```

Comment:
useless, ```login``` is already ahndle after (when we are not in a context of anonymization)
just the display part was missing